src/Output/Output_DumpData_Total_HDF5.cpp

```diff
@@ -1588,6 +1589,96 @@ void FillIn_KeyInfo( KeyInfo_t &KeyInfo, const int NFieldStored )
    srand( time(NULL) );
    KeyInfo.UniqueDataID = rand();
 
+// ugly index from Aux_Check_Conservation.cpp
+#  if   ( MODEL == HYDRO )
+#  ifdef MHD
+   const int    NVar_NoPassive    = 12;
+#  else
+   const int    NVar_NoPassive    = 11;
+#  endif
+   const int    idx_etot          = NVar_NoPassive - 1;
+#  elif ( MODEL == ELBDM )
+   const int    NVar_NoPassive    = 11;
+   const int    idx_etot          = NVar_NoPassive - 1;
+#  endif
+   const bool GetPassiveSum = ( PassiveNorm_NVar > 0 );
+   const int  NVar          = NVar_NoPassive + NCOMP_PASSIVE + ( (GetPassiveSum)?1:0 );
+
+   if ( !ConservedRefLoaded )   Aux_Error( ERROR_INFO, "Conserved variables reference are not assigned yet!!\n" );
+   KeyInfo.Time_H5Ref        = Time_Ref;
+#  if   ( MODEL == HYDRO )
+   KeyInfo.Mass_Gas_H5Ref    = Fluid_Ref[0];
```

Comment:
I still think we should include keywords related to "conservation" or "total" in these variable names. Otherwise, it is hard to understand the type of reference when someone looks at the hdf5 data.